inc/mailcollector.class.php

```diff
@@ -308,11 +308,11 @@ function showForm($ID, $options = []) {
       Dropdown::showYesNo("use_mail_date", $this->fields["use_mail_date"]);
       echo "</td></tr>\n";
 
-      echo "<tr class='tab_bg_1'><td>" . __('Field for requester') . "</td>";
+      echo "<tr class='tab_bg_1'><td>" . __('Use Reply-To when available') . "</td>";
```

Comment:
This is not really clear. The label does not say "for what" the reply-to field will be used.

Reply:
I've improved the label.